keel-sql/src/main/resources/db/changelog/20200928-add-notifier-repo.yml

```
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-notifier-repo
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: notifier
```

Please change the table name to match the class:
```suggestion
            tableName: notifications
```

---

ahhhh I missed that one, thanks!